bin/rebuild

```diff
@@ -31,6 +32,38 @@ versiondb_head() {
   (cd "$VERSIONDB" && git rev-parse HEAD)
 }
 
+save_build_env() {
+  # saving environment information
+  filename="${LSSTSW_BUILD_DIR}/${BUILD}.env"
+  echo "Saving environment information in ${filename}"
+  conda env export | grep -v "^name"> /tmp/lenv
+  echo "environment_name: ${LSST_CONDA_ENV_NAME}" > "${filename}"
+  cat /tmp/lenv | grep "^prefix" >> "${filename}"
+  cat /tmp/lenv | grep -v "^prefix" > /tmp/lenv1
+  if [[ "${LSST_CONDA_ENV_NAME}" != "lsst-scipipe" ]]; then
+    ref="${LSST_CONDA_ENV_NAME:12}"
+    if [[ $ref == *"."* ]]; then
+      ref1=`echo "${ref}" | cut -f 1 -d .`
+      ref2=`echo "${ref}" | cut -f 2 -d .`
+      local_env_dir="${LSSTSW}/env/${ref1}/${ref2}"
+    else
+      local_env_dir="${LSSTSW}/env/${ref}"
+    fi 
+    if [ -d "${local_env_dir}" ]; then
+      ref_file=`ls "${local_env_dir}"`
```

Comment:
This seems dangerous.  You should check explicitly for a `conda*.yml` file, not just blindly assume any file (and what if there's zero or more than one?) in the env directory is appropriate.